e2e-tests/content-strings/error-messages.js

```diff
@@ -245,7 +257,7 @@ export const ERROR_MESSAGES = {
             IS_EMPTY: 'Enter your last name',
           },
           [FIELD_IDS.INSURANCE.ACCOUNT.EMAIL]: {
-            INCORRECT_FORMAT: 'Enter your email address in the correct format - for example name@example.com',
+            INCORRECT_FORMAT: 'Enter your email address in the correct format, like name@example.com',
```

Comment:
Whilst you're in here could you do a search for "in the correct format" and make sure they're aligned please? Would fix this small bug https://ukef-dtfs.atlassian.net/browse/EMS-1541